core/websock.js

```diff
@@ -45,58 +45,58 @@ export default class Websock {
     }
 
     // Getters and Setters
-    get_sQ() {
+    get sQ() {
         return this._sQ;
     }
 
-    get_rQ() {
+    get rQ() {
         return this._rQ;
     }
 
-    get_rQi() {
+    get rQi() {
         return this._rQi;
     }
 
-    set_rQi(val) {
+    set rQi(val) {
         this._rQi = val;
     }
 
     // Receive Queue
-    rQlen() {
+    get rQlen() {
         return this._rQlen - this._rQi;
     }
 
     rQpeek8() {
         return this._rQ[this._rQi];
     }
 
-    rQshift8() {
-        return this._rQ[this._rQi++];
-    }
-
-    rQskip8() {
-        this._rQi++;
+    rQskipBytes(bytes) {
+        this._rQi += bytes;
     }
 
-    rQskipBytes(num) {
-        this._rQi += num;
+    rQshift8() {
+        return this.rQshift(1);
     }
 
-    // TODO(directxman12): test performance with these vs a DataView
     rQshift16() {
-        return (this._rQ[this._rQi++] << 8) +
-               this._rQ[this._rQi++];
+        return this.rQshift(2);
```

Comment:
These changes should probably be a separate commit as they don't really match the commit message.

Reply:
Do you mean extracting to a common function?